src/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { IGridLayout } from './state/interfaces/controls';
+
+/**
+ * Offers constant information values to use in an application.
+ */
+export module InteractiveConstants {
```

Comment:
I figured that if there's more constants stuff to be exported in the future, you'd wanna wrap it anyways 😊
Makes it a bit cleaner in my opinion.